release.sh

```diff
@@ -1,6 +1,18 @@
 #!/bin/bash -e
 # Releases this Puppet module to the Puppet Forge
 
+# Check to see if the current version is already in Puppet Forge
+FORGE_MODULE_URL='https://forgeapi.puppet.com/v3/modules/cyberark-conjur'
+FORGE_MODULE_VERSION="$(curl --silent ${FORGE_MODULE_URL} \
+                          | jq -r '.current_release.version')"
+LOCAL_MODULE_VERSION="$(jq -r '.version' metadata.json)"
+
```

Comment:
You can `curl https://forgeapi.puppet.com/v3/releases/cyberark-conjur-$LOCAL_MODULE_VERSION` instead to check if a given release exists. Checking `current_release.version` will give a false negative if the local version is on forge but isn't the latest.